examples/k8s-bookinfo.yaml

```diff
@@ -56,8 +56,10 @@ spec:
           value: http://$(CONTROLLER_SERVICE_HOST):$(CONTROLLER_SERVICE_PORT)
         - name: A8_DISCOVERY_BACKEND
           value: kubernetes
-        - name: A8_SERVICE
-          value: gateway:none
+        - name: A8_KUBERNETES_POD_NAME
```

Comment:
I've removed the `:none` tag from gremlin JSONs, based on the `example-*.json` files in the [amalgam8/gremlin](https://github.com/amalgam8/gremlin).